### ISSUE TITLE
Can't drag inventory items from sheet to foundry items

### ISSUE THREAD

**Issue by GregorHak** [DrOgres/twdu#47]
Can't drag inventory items from sheet to foundry items, see video bellow

https://github.com/user-attachments/assets/76fd4ae4-504a-4bfd-8d42-84d832b09394

**Comment by DrOgres** [comment]
Yeah, not 100 on how to implement that will investigate. When I get a spare moment.
